katello/helper.rb

```diff
@@ -13,7 +13,7 @@ def error_message
     end
 
     def foreman_rpm_installed?
-      system("rpm -q foreman")
+      system("rpm -q foreman >/dev/null")
```

Comment:
Is this change relevant here?

Reply:
It prints the output needlessly, and since we aren't doing anything except distinguishing satellite from capsule, I don't think the output created should be displayed.

Reply:
fair enough.

Reply:
this has been fixed via http://projects.theforeman.org/issues/21219 now